core/src/sumeragi/mod.rs

```diff
@@ -883,7 +883,7 @@ impl<G: GenesisNetworkTrait, K: KuraTrait, W: WorldTrait> Sumeragi<G, K, W> {
             if peer.address == self_address || peers_online.contains(&peer.public_key) {
                 continue;
             }
-            iroha_logger::info!("Connecting {}", &peer.address);
+            iroha_logger::info!(peer_addr = %peer.address, "Connecting peer");
```

Comment:
```suggestion
            iroha_logger::info!(%peer.address, "Connecting peer");
```
The shorthand notation extends to fields of fields.